src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -1321,6 +1321,22 @@ class DotVVM {
             }
         };
 
+        ko.bindingHandlers["dotvvm-checkedItems"] = {
+            after: ko.bindingHandlers.checked.after,
+            init: ko.bindingHandlers.checked.init,
+            options: ko.bindingHandlers.checked.options,
+            update: function (element, valueAccessor, allBindings, viewModel, bindingContext) {
+                var value = valueAccessor();
+                if (!Array.isArray(valueAccessor()) && (!ko.isObservable(value) || !Array.isArray(value()))) {
```

Comment:
Is this code equivalent to ``Array.isArray(ko.unwrap(value))`, just more ... intriguing?